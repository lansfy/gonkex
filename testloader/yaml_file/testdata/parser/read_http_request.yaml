- name: simple request test
  description: simple request test description
  method: POST
  headers:
    X-Request-Header: header_value_1
  cookies:
    somecookie1: someid1
    somecookie2: someid1
  query: ?param=value
  path: /some/path
  request: "requestbody"
  response:
    200: "aaabbb"
  responseHeaders:
    200:
      X-Header: header_value_2
