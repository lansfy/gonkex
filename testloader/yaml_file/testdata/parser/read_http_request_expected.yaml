- GetName: simple request test
  GetDescription: simple request test description
  GetMethod: POST
  Headers:
    X-Request-Header: header_value_1
  Cookies:
    somecookie1: someid1
    somecookie2: someid1
  Path: /some/path
  ToQuery: ?param=value
  GetRequest: requestbody
  GetResponses:
    200: "aaabbb"
  GetFileName: testdata/parser/read_http_request.yaml
  FirstTestInFile: true
  LastTestInFile: true
